samtranslator/policy_templates_data/policy_templates.json

```diff
@@ -1253,7 +1253,10 @@
       "Definition": {
         "Statement": [
           {
-            "Action": "kms:Encrypt",
+            "Action": [
+              "kms:Encrypt",
+              "kms:GenerateDataKey"
+            ],
```

Comment:
This will break backwards compatibility for people using `KMSEncryptPolicy` unfortunately. We can create a new policy called `KMSEncryptPolicy_v2` with the new permissions. Example PR you can follow: https://github.com/aws/serverless-application-model/pull/2929

Reply:
My change is backward compatible because I'm only adding a new permission, not removing anything.

Reply:
Will double check this with another team member but I believe adding a new permissions will also be backwards incompatible. If a user has a SAM Function with `KMSEncryptPolicy` attached then the transformed template will have role with 
```
PolicyDocument": {
       "Statement": [
        {
         "Action": "kms:Encrypt",
         "Effect": "Allow",
         "Resource": {
          ...
         }
```
with your change the transformed template will be
```
PolicyDocument": {
       "Statement": [
        {
         "Action": [
          "kms:Encrypt",
          "kms:GenerateDataKey"
         ],
         "Effect": "Allow",
         "Resource": {
          ...
         }
```
The user didn't make any changes to their SAM template but the CFN template change which causes re-deployments which we consider backwards incompatible